experiments/process-kill/experiment/process-kill.go

```diff
@@ -36,22 +36,28 @@ func ProcessKill(w http.ResponseWriter, r *http.Request) {
 	executeExperimentErrorLogger := logger.GetExecuteExperimentErrorLogger()
 	commandProbeExecutionErrorLogger := logger.GetCommandProbeExecutionErrorLogger()
 	invalidActionErrorLogger := logger.GetCommandProbeExecutionErrorLogger()
+	livenessCheckErrorLogger := logger.GetLivenessCheckErrorLogger()
+	closeConnectionErrorLogger := logger.GetCloseConnectionErrorLogger()
 
 	conn, err := upgrader.Upgrade(w, r, nil)
 	if err != nil {
-		log.Printf("Failed to establish connection with client, %v", err)
+		fmt.Fprintf(w, "Failed to establish connection with client, err: %v", err)
```

Comment:
Any specific reason as to why changing to fmt lib from log lib for logging?

Reply:
We are not supposed to log here anything, it was a mistake which I am rectifying this time. `fmt.Fprintf` is supposed to write the error message to the ResponseWriter object `w`, so that it can be sent back to the client as part of the HTTP request only (since we were unable to upgrade to WebSocket).